tests/unit/test_dataset_builders/test_contents_define_vlm_dataset_builder.py

```diff
@@ -80,4 +82,5 @@ def test_contents_define_vlm_dataset_builder(
     expected_df = pd.DataFrame.from_dict(expected)
     expected_df.sort_index(axis=1, inplace=True)
     result.sort_index(axis=1, inplace=True)
+    assert "define_vlm_mandatory" in result
```

Comment:
we should check the value matches as well and not just the existence of the attribute